arcade/sprite/__init__.py

```diff
@@ -66,6 +66,5 @@ def load_animated_gif(resource_name) -> AnimatedTimeBasedSprite:
     "FACE_RIGHT",
     "FACE_UP",
     "FACE_DOWN",
-    "PymunkMixin",
-    "Pymunk",
```

Comment:
There is a `PyMunk` imported at the top of this file, which this line may have been a misspelled reference to. Can you double check this?

Reply:
Yeah, github is hiding it, but `"PyMunk"` actually appears earlier in this list. Come to look at it again, there's also `"SimpleSprite"` commented out, not sure why that is?

Reply:
I thought I removed that line, but it seems you're going to be the one to do it. To my understanding, it refers to what became `BasicSprite`.

Reply:
Nice, gimme those sweet sweet git blame points.